From 92d06e843c4d105175e2e1961f94ce879f84630f Mon Sep 17 00:00:00 2001
From: David Hildenbrand <david@redhat.com>
Date: Tue, 17 Oct 2017 19:15:50 +0200
Subject: [PATCH 45/69] tools/kvm_stat: remove extra statement

RH-Author: David Hildenbrand <david@redhat.com>
Message-id: <20171017191605.2378-25-david@redhat.com>
Patchwork-id: 77332
O-Subject: [RHEL-7.5 qemu-kvm-rhev PATCH 24/39] tools/kvm_stat: remove extra statement
Bugzilla: 1497137
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

Upstream-status: linux.git 5e3823a49c50d70cc6b92808c262a43cf3505f3c

commit 5e3823a49c50d70cc6b92808c262a43cf3505f3c
Author: Stefan Raspl <raspl@linux.vnet.ibm.com>
Date:   Wed Jun 7 21:08:31 2017 +0200

    tools/kvm_stat: remove extra statement

    Signed-off-by: Stefan Raspl <raspl@linux.vnet.ibm.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 scripts/kvm/kvm_stat | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/kvm/kvm_stat b/scripts/kvm/kvm_stat
index 1b8626b..e38791d 100755
--- a/scripts/kvm/kvm_stat
+++ b/scripts/kvm/kvm_stat
@@ -1066,7 +1066,6 @@ class Tui(object):
 
             except ValueError:
                 msg = '"' + str(pid) + '": Not a valid pid'
-                continue
 
     def show_vm_selection_by_guest_name(self):
         """Draws guest selection mask.
-- 
1.8.3.1

